.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: csharp
 
 os: osx
-osx_image: xcode9.3
+osx_image: xcode10
```

Comment:
Was this change needed?

It looks like it changes the version of Java that is installed

1.8.0 https://travis-ci.org/fsprojects/Fabulous/builds/433555917#L70

vs

10.0.2 https://travis-ci.org/fsprojects/Fabulous/builds/433529887#L59

Reply:
I don't think so.
Changed it as part of the global update (along with Xamarin.iOS and Xamarin.Android)
But it can be reverted if needed.

Reply:
Looks I came to the same conclusion independently [from you](https://github.com/fsprojects/Fabulous/pull/196#issuecomment-424747145) :)

Looks like reverting this change is the simplest solution providing everything else works.